### PR TITLE
Navigation screen

### PR DESCRIPTION
**Pull request by Shaktiexcellence**

### CODE REVIEW

File: lib/screens/navigate/inventory.dart [hunk 1]
```diff
@@ -8,16 +9,23 @@ import '../navigate/recent_comments.dart';
...
   _MyInventoryState createState() => _MyInventoryState();
 }
 
 class _MyInventoryState extends State<MyInventory> {
+  int _index=0;  
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
+    if(Platform.isIOS){
+
```
Comment (flutterexcel) [inline lib/screens/navigate/inventory.dart]: @Shaktiexcellence why this code

File: lib/screens/navigate/inventory.dart [hunk 2]
```diff
@@ -30,13 +38,18 @@ class _MyInventoryState extends State<MyInventory> {
...
       drawer: Navigation(),
-      floatingActionButton: FloatingActionButton.extended(
-        onPressed: () {
-          // Add your onPressed code here!
-        },
-        label: Text('Assign Device'),
-        backgroundColor: AppColors.GREEN_COLOR,
-      ),
+      
+      floatingActionButton: CupertinoButton (
```
Comment (flutterexcel) [inline lib/screens/navigate/inventory.dart]: @Shaktiexcellence where ios condition

File: lib/screens/navigate/my_inventory.dart [hunk 3]
```diff
@@ -0,0 +1,154 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:universal_io/io.dart';
+import '../navigate/navigate.dart';
+import 'package:flutter_excellence_hr/widgets/navigate/imp_notes_inventory.dart';
+import 'package:flutter_excellence_hr/resources/app_colors.dart';
+import 'package:flutter_excellence_hr/widgets/navigate/comment_section.dart';
+
+class YourInventory extends StatelessWidget {
+  @override
```
Comment (flutterexcel) [inline lib/screens/navigate/my_inventory.dart:10]: @Shaktiexcellence change class to proper name

File: lib/screens/navigate/my_inventory.dart [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+  Widget build(BuildContext context) {
+    if (Platform.isIOS) {
+      print('OS: ${Platform.isIOS}');
+    }
+ customDialog() {
```
Comment (flutterexcel) [inline lib/screens/navigate/my_inventory.dart:15]: @Shaktiexcellence it need to be seperate widget

File: lib/screens/navigate/my_inventory.dart [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+                        'Device Details',
+                        style: TextStyle(
+                          color: AppColors.LIGHTBLACK_COLOR,
+                          fontSize: 25,
+                          fontWeight: FontWeight.bold,
+                        ),
+                      ),
+                    ),
+                    DropDown(),
+                    InventoryItemDetails(
```
Comment (flutterexcel) [inline lib/screens/navigate/my_inventory.dart:42]: @Shaktiexcellence all Inventory item details need to be listviewbuilder

File: lib/screens/navigate/my_inventory.dart [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+                    fontSize: 20,
+                    fontWeight: FontWeight.bold)),
+            CircleAvatar(
+                radius: 20,
+                backgroundImage: AssetImage('assets/images/person.jpg'))
+          ],
+        ),
+      ),
+      drawer: Navigation(),
+      floatingActionButton: CupertinoButton(
```
Comment (flutterexcel) [inline lib/screens/navigate/my_inventory.dart:97]: @Shaktiexcellence where's ios condition and where is material for ios

File: lib/screens/navigate/inventory.dart [hunk 7]
```diff
@@ -16,90 +17,74 @@ class MyInventory extends StatefulWidget {
...
+        child: SingleChildScrollView(
+          child: Column(
+            children: <Widget>[
+              ImportantNotesInventory(),
+              SizedBox(height: 20),
+              InkWell(
+                onTap: () {
+                  customDialog();
+                },
+                child: InventoryItems(
```
[the remark this answers is not in the document]
Comment (Shaktiexcellence) [reply lib/screens/navigate/inventory.dart]: used SingleChildScrollView didn't use any ListView

File: lib/screens/navigate/navigation.dart [hunk 8]
```diff
@@ -1,11 +1,11 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_excellence_hr/resources/app_colors.dart';
+import 'package:flutter_excellence_hr/screens/navigate/inventory.dart';
 
 void main() => runApp(Navigation());
```
[the remark this answers is not in the document]
Comment (Shaktiexcellence) [reply lib/screens/navigate/navigation.dart:5]: removed void main

File: lib/screens/navigate/my_inventory.dart (hunk 3 above)
Comment (Shaktiexcellence) [reply lib/screens/navigate/my_inventory.dart:10]: changed

File: lib/screens/navigate/inventory.dart [hunk 9]
```diff
@@ -151,12 +137,12 @@ class _MyInventoryState extends State<MyInventory> {
                   SizedBox(
                     height: 30,
                   ),
-                  for(int i=0;i<5;i++)
-                  CommentSection(
-                    name: 'Deepak Mishra',
-                    date: '6th April20, 11:55 am',
-                    comment: 'All Good',
-                  ),
+                  for (int i = 0; i < 5; i++)
```
[the remark this answers is not in the document]
Comment (Shaktiexcellence) [reply lib/screens/navigate/inventory.dart]: Done it

File: lib/screens/navigate/my_inventory.dart (hunk 5 above)
Comment (Shaktiexcellence) [reply lib/screens/navigate/my_inventory.dart:42]: Done it In ListView Builder

File: lib/screens/navigate/my_inventory.dart (hunk 6 above)
Comment (Shaktiexcellence) [reply lib/screens/navigate/my_inventory.dart:97]: IOS condition is written only once in Application , in Home page to return CupertinoApp or MaterialApp

**Pull request merged**